### PR TITLE
get the weather condition and imaes show according to the weather

### PR DESCRIPTION
**Pull request by husamkamal**
Relates #12

### CODE REVIEW

File: js/home.js [hunk 1]
```diff
@@ -9,204 +9,247 @@ weekInfoContainer.className = "container";
...
+            lat = weatherInfo.coord.lat;
+            lon = weatherInfo.coord.lon;
+            countryName = weatherInfo.sys.country;
+            cityName = weatherInfo.name;
+            fetch(
+                    `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=minutely,hourly,alerts&appid=${apiKey}`
+                )
+                .then((info) => info.json())
+                .then((db) => {
+                    console.log(db)
```
Comment (sara219) [inline js/home.js]: remove any unused console.log that you used for testing

File: js/home.js [hunk 2]
```diff
@@ -9,204 +9,247 @@ weekInfoContainer.className = "container";
...
+                            Windy: ["../assets/images/windy/1.jpg", "../assets/images/windy/2.jpg", "../assets/images/windy/3.jpg", "../assets/images/windy/4.jpg"]
+                        }
+                        //Compare the weather with the key of object
+                    let describ = "";
+                    switch (true) {
+                        case currente == "Clouds":
+                            describ = images.Clouds
+                            break;
+                        case currente == "Rain":
+                            describ = images.Rain
```
Comment (sara219) [inline js/home.js]: use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to format your code so the page will look neat

**Review by sara219 — CHANGES_REQUESTED** [review]
I tried to search for a country that may not exist or I wrote its name incorrectly, and the result of the previous search remained without any warning to the user that an error occurred, despite the presence of an error 404 in the console

do you have a plan to fix this error in the future or its didn't come to your mind for now!

File: js/home.js [hunk 3]
```diff
@@ -9,204 +9,247 @@ weekInfoContainer.className = "container";
...
+                )
+                .then((info) => info.json())
+                .then((db) => {
+                    console.log(db)
+                        ////////////////////////////
+                    let currente = db.current.weather[0].main //give the weather condition 
+                        //images show According to the weather 
+                    let images = {
+                            Rain: ["../assets/images/rainy/1.jpg", "../assets/images/rainy/2.jpg", "../assets/images/rainy/3.jpg", "../assets/images/rainy/4.jpg"],
+                            Clear: "../assets/images/clear/1.jpg",
```
Comment (zaher-aa) [inline js/home.js]: Put `"../assets/images/clear/1.jpg"` inside an array, so the `randomImages()` function can choose a random image from the array

**Review by zaher-aa — CHANGES_REQUESTED** [review]

**Comment by zaher-aa** [comment]
> I tried to search for a country that may not exist or I wrote its name incorrectly, and the result of the previous search remained without any warning to the user that an error occurred, despite the presence of an error 404 in the console
> 
> do you have a plan to fix this error in the future or its didn't come to your mind for now!

We have opened an issue for this case and Hussam assigned himself to it.
He'll work on it after finishing his current task.

**Comment by husamkamal** [comment]
> !

The problem of the lack of the state was planned to be solved and there is a special isusse in it. As for the problem of writing the name of the state incorrectly, there is an autocomplete, but I have not uploaded it yet

**Comment by husamkamal** [comment]
The problem of the lack of the state was planned to be solved and there is a special isusse in it. As for the problem of writing the name of the state incorrectly, there is an autocomplete, but I have not uploaded it yet

**Review by zaher-aa — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**